frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -380,9 +381,14 @@ const AlertManagerConfiguration: React.FC<AlertManagerConfigurationProps> = ({ o
 export const AlertManagerConfigWrapper: React.FC<AlertManagerConfigWrapperProps> = React.memo(
   ({ obj, ...props }) => {
     return (
-      <StatusBox {...obj}>
-        <AlertManagerConfiguration {...props} obj={obj.data} />
-      </StatusBox>
+      <>
+        <Helmet>
+          <title>Alertmanager Overview</title>
```

Comment:
We should match the page heading.

```suggestion
          <title>Alerting</title>
```